gluoncv/utils/filesystem.py

```diff
@@ -21,7 +21,7 @@ def makedirs(path):
         if exc.errno != errno.EEXIST:
             raise
 
-def try_import(package, message=None):
+def try_import(package, fromlist=[], message=None):
```

Comment:
Can we set the default number to None? It is not a good coding style to set it to [].